### PR TITLE
Generation of candidates

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/w3c/webrtc-pc/issues/1050

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -83,6 +83,7 @@
     <dfn>MediaStreamConstraints</dfn> are defined in [[!GETUSERMEDIA]].</p>
     <p>The term <dfn>Blob</dfn> is defined in [[!FILEAPI]].</p>
     <p>The term <dfn>media description</dfn> is defined in [[!RFC4566]].</p>
+    <p>The term <dfn>generation</dfn> is defined in [[TRICKLE-ICE]] Section 2.</p>
```
Comment (taylor-b) [inline webrtc.html]: "[[!TRICKLE-ICE]]" to make this a normative reference?

Comment (aboba) [inline webrtc.html]: @stefhak Do you know why the reference to Trickle-ICE is non-normative? Isn't this mandatory-to-implement??

Comment (alvestrand) [inline webrtc.html]: Seems logical to make it normative.

**Comment by alvestrand** [comment]
LGTM

**Pull request merged**